VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -142,6 +144,7 @@ public async Task<IActionResult> GetIndependentTestCallResultAsync(Guid particip
             }
         }
 
+        [ServiceFilter(typeof(CheckParticipantCanAccessConferenceAttribute))]
         [HttpGet("{conferenceId}/participant/{participantId}/heartbeatrecent")]
         [SwaggerOperation(OperationId = "GetHeartbeatDataForParticipant")]
         [ProducesResponseType(typeof(ParticipantHeartbeatResponse[]), (int)HttpStatusCode.OK)]
```

Comment:
`[Authorize(AppRoles.VhOfficerRole)]` might be better there rather than the service filter since only VHOfficers use this endpoint

Reply:
The filter is applied globally, and I think this particular endpoint can used by any participant.

Reply:
I've added the [Authorize] filter now.